src/Indice.Features.Cases.App/src/app/features/cases/case-detail-page/case-form/case-form.component.ts

```diff
@@ -140,39 +140,58 @@ export class CaseFormComponent implements OnChanges, OnInit, OnDestroy {
           }
         }
       }
-      forkJoin(callsDict)
-        .pipe(
-          tap(
-            // we can match the received attachment guids to the respective fields with the help of the keys/dataPointers!
-            (response: { [key: string]: CasesAttachmentLink }) => {
-              let stringifiedData = JSON.stringify(event);
-              for (const key in response) {
-                if (response.hasOwnProperty(key)) {
-                  // we simply replace the dataPointer with the guid that we received from the server
-                  stringifiedData = stringifiedData.replace(key, response[key].id!);
-                }
-              }
-              // finally, submit the case
-              this._api.submitAdminCase(this.case?.id!, stringifiedData)
-                .pipe(
-                  tap(() => {
-                    this._fileUploadService.reset();
-                    this._toaster.show(ToastType.Success, 'Επιτυχής Επεξεργασία', `Η επεξεργασία της αίτησης ολοκληρώθηκε.`, 5000);
-                    this.updateDataEvent.emit({ draft: true });
-                  }),
-                  catchError((err: ProblemDetails) => { // error during case submit
-                    this._toaster.show(ToastType.Error, 'Αποτυχία αποθήκευσης', err.detail || `Δεν κατέστη εφικτή η καταχώριση της αίτησης σας.`, 5000);
-                    this.router.navigate(['/cases']);
-                    return EMPTY;
-                  })
-                )
-                .subscribe();
+      if (Object.keys(callsDict).length === 0) { // there are no documents to be uploaded!
```

Comment:
Δεν μπορώ να σχολιάσω κώδικα που δεν έχει αλλαχτεί, αλλά αν κάνεις στην original line 129 ` const callsDict: { [key: string]: Observable<CasesAttachmentLink> } = { '': of({} as CasesAttachmentLink) };`

και μετά στην original line 150 `if (response.hasOwnProperty(key) && key !== '') {`

θα καταφέρεις το ίδιο πράγμα.

Στην ουσία αρχικοποιείες με default value `''` το οποίο. κατόπιν, αγνοείς